eng/Compilers.props

```diff
@@ -6,7 +6,17 @@
     <CompilerVersion>3.6.0-2.20166.2</CompilerVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="!($(MSBuildProjectName.EndsWith('Tests', StringComparison.OrdinalIgnoreCase)))">
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn></NoWarn> <!-- Make sure the list is empty - by default it is not -->
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
NIT: this is always true so let's just define it in an unconditioned propertyGroup instead.